_src/pattern-library/sass/utilities/_variables.scss

```diff
@@ -141,55 +141,58 @@ $shadow: transparentize(palette(grayscale, black-t), .5);
 // ----------------------------
 // #TYPOGRAPHY
 // ----------------------------
+
+$ux-pattern-library-font-path: '../fonts' !default;
```

Comment:
I understand it is not the intended use case; however, until someone recommends a better implementation, it is the world we live in. Allowing the pattern library to be used free of potential conflicts eases the burden on our team as well as every other team that will need to consider a similar integration as we continue to break apart edx-platform. We are talking about a single variable, not an exception to the entire project. There is no slippery slope here.